code/modules/jobs/job_types/shaft_miner.dm

```diff
@@ -30,6 +62,16 @@
 	job_flags = JOB_ANNOUNCE_ARRIVAL | JOB_CREW_MANIFEST | JOB_EQUIP_RANK | JOB_CREW_MEMBER | JOB_NEW_PLAYER_JOINABLE | JOB_REOPEN_ON_ROUNDSTART_LOSS | JOB_ASSIGN_QUIRKS | JOB_CAN_BE_INTERN
 
 
+/datum/job/shaft_miner/after_spawn(mob/living/spawned, client/player_client)
+	..()
+	RegisterSignal(spawned, COMSIG_MOB_POINTED, PROC_REF(point_speech))
+
+/datum/job/shaft_miner/proc/point_speech(mob/pointing_miner, atom/movable/object_of_interest)
+	SIGNAL_HANDLER
```

Comment:
also needs a cooldown, 10 seconds or something will be fine

Reply:
Was about to say, either a cooldown, or a recharge with a pool of automatic call outs so you could have a few stocked up, but not enough to spam the chat (EG: You cap at 3 callout charges based on callbacks or something similar.)